AdminWebsite/AdminWebsite/Controllers/HearingsController.cs

```diff
@@ -455,30 +451,60 @@ public async Task<ActionResult> GetHearingById(Guid hearingId)
                 catch (VideoApiException ex)
                 {
                     _logger.LogError(ex, "Failed to confirm a hearing. {ErrorMessage}", errorMessage);
-                }
-
-                _logger.LogError("There was an unknown error for hearing {Hearing}. Updating status to failed",
+                    _logger.LogError("There was an unknown error for hearing {Hearing}. Updating status to failed",
                     hearingId);
-                // Set the booking status to failed as the video api failed
-                await _bookingsApiClient.UpdateBookingStatusAsync(hearingId,
-                    new UpdateBookingStatusRequest
-                    {
-                        Status = BookingsApi.Contract.Requests.Enums.UpdateBookingStatus.Failed,
-                        UpdatedBy = "System",
-                        CancelReason = string.Empty
-                    });
-                return Ok(new UpdateBookingStatusResponse { Success = false, Message = errorMessage });
+
+                    // Set the booking status to failed as the video api failed
+                    await _bookingsApiClient.UpdateBookingStatusAsync(hearingId,
+                        new UpdateBookingStatusRequest
+                        {
+                            Status = BookingsApi.Contract.Requests.Enums.UpdateBookingStatus.Failed,
+                            UpdatedBy = "System",
+                            CancelReason = string.Empty
+                        });
+
+                    return Ok(new UpdateBookingStatusResponse { Success = false, Message = errorMessage });
+                }
             }
-            catch (BookingsApiException e)
+            catch (Exception ex)
             {
-                if (e.StatusCode == (int)HttpStatusCode.BadRequest) return BadRequest(e.Response);
-                if (e.StatusCode == (int)HttpStatusCode.NotFound) return NotFound(e.Response);
-                _logger.LogError(e, "There was an unknown error updating status for hearing {Hearing}", hearingId);
-                throw;
+                _logger.LogError(ex, "There was an unknown error updating status for hearing {Hearing}", hearingId);
+                if (updateBookingStatusRequest.Status == BookingsApi.Contract.Requests.Enums.UpdateBookingStatus.Created)
+                {
+                    // Set the booking status to failed as the video api failed
+                    await _bookingsApiClient.UpdateBookingStatusAsync(hearingId,
+                        new UpdateBookingStatusRequest
+                        {
+                            Status = BookingsApi.Contract.Requests.Enums.UpdateBookingStatus.Failed,
+                            UpdatedBy = "System",
+                            CancelReason = string.Empty
+                        });
+
+                    return Ok(new UpdateBookingStatusResponse { Success = false, Message = errorMessage });
+                }
+                if (ex is BookingsApiException)
```

Comment:
can we chain this exception instead of adding an if logic here: 
```
catch (BookingsApiException e)
{
//implementation
}
 catch (Exception ex)
{
//implementation
}
```

Reply:
you will have duplicated lines of code the booking service for the bad request of we do like this.

Reply:
I had thought thur this and I think we will be duplicating code in each catch, the code structure was previously not that good, so to avoid refactoring we have to live with this.